pandas/tests/test_frame.py

```diff
@@ -5545,6 +5545,8 @@ def test_itertuples(self):
         dfaa = df[['a', 'a']]
         self.assertEqual(list(dfaa.itertuples()), [(0, 1, 1), (1, 2, 2), (2, 3, 3)])
 
+        self.assertEqual(type(next(df.itertuples(name=None))), tuple)
```

Comment:
Switched to `repr` checking. Note that grepping `tests` for `Equal(type(` there are some more of such explicit type comparisons (which IMHO this is fine).